src/main/java/org/myrobotlab/i2c/I2CProxyDeviceImpl.java

```diff
@@ -81,4 +83,16 @@ public int getAddress() {
     return 0;
   }
 
+  @Override
+  public void ioctl(long arg0, int arg1) throws IOException {
+    // TODO Auto-generated method stub
+    
+  }
+
+  @Override
+  public void ioctl(long arg0, ByteBuffer arg1, IntBuffer arg2) throws IOException {
+    // TODO Auto-generated method stub
+    
+  }
+

```

Comment:
are these required from interface changes ?

Reply:
not required to get servo worky with wiringPi, but not sure at all about the rest of the features... a raspi guru will look at this certainly in the future